common/powershell/hyperv/hyperv.go

```diff
@@ -617,9 +617,9 @@ foreach ($adapter in $adapters) {
   }
 }
 
-if($switch -ne $null) { 
-  Get-VMNetworkAdapter -VMName $vmName | Connect-VMNetworkAdapter -VMSwitch $switch 
-} else { 
+if($switch -ne $null) {
+  Get-VMNetworkAdapter -VMName $vmName | Connect-VMNetworkAdapter -VMSwitch $switch
```

Comment:
I'm also hesitant to change the indentation of this, since I'm not as familiar with how powershell handles whitespace. Maybe it's important?